### PR TITLE
updated RAM overview

### PR DESCRIPTION
**Pull request by jerae-duffin**
Added an example to overview section

### CODE REVIEW

File: docker-hub/registry-access-management.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ keywords: registry, access, managment
 title: Registry Access Management
 ---
 
-Registry Access Management is a feature available to organizations with a Docker Business subscription. With Registry Access Management, organization owners can ensure that their developers using Docker Desktop can only access registries that have been allow-listed via the Registry Access Management dashboard on Docker Hub to reflect support for other registries: AWS ECR, GitHub Container Registry, Google Container Registry, Quay, and others.
+Registry Access Management (RAM) is a feature available to organizations with a Docker Business subscription. When RAM is enabled, organization owners can ensure that their developers using Docker Desktop can only access registries that have been allow-listed via the Registry Access Management dashboard on Docker Hub to reflect support for other registries: AWS ECR, GitHub Container Registry, Google Container Registry, Quay, and others.
```
Comment (djs55) [inline docker-hub/registry-access-management.md:7]: (just a suggestion) Maybe while editing this text we could add "a local registry" as an example? So perhaps
> AWS ECR, GitHub Container Registry, Google Container Registry, Quay, a local private registry, and others.

File: docker-hub/registry-access-management.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ keywords: registry, access, managment
...
+
+For example, you can use RAM if you manage engineering teams that use Docker Desktop for local deployment and want to ensure that the images they are pulling are licensed and reputable before using them.
```
Comment (djs55) [inline docker-hub/registry-access-management.md:9]: I think "use Docker Desktop for local development" ("development" rather than "deployment")

**Review by djs55 — APPROVED** [review]
A few minor suggestions, otherwise LGTM

File: docker-hub/registry-access-management.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/registry-access-management.md:7]: Updated, thank you!

File: docker-hub/registry-access-management.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/registry-access-management.md:9]: Updated

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**